RiotSwiftUI/Modules/Room/Composer/Model/ComposerViewState.swift

```diff
@@ -19,12 +19,13 @@ import Foundation
 struct ComposerViewState: BindableState {
     var eventSenderDisplayName: String?
     var sendMode: ComposerSendMode = .send
+    var textFormattingEnabled = true
```

Comment:
@aringenbach had to remove the RiotSettings reference from here since it's not included in RiotSwiftUI code, and prevented the build to happen and complete. It will naturally be set externally, so this should not be an issue.